content/SCALE/SCALETutorials/Apps/EnterpriseApps/Syncthing.md

```diff
@@ -51,122 +60,134 @@ Accept the default values in **Application Name** and **Version**.
 
-{{< trueimage src="/images/SCALE/Apps/InstallSyncthingEnterpriseStorageConfigIxVolume.png" alt="Syncthing Storage Add ix-Volume" id="Syncthing Storage Add ix-Volume" >}}
+Next, Click **add** to the right of **Additional Storage** to add the storage configuration settings for the data volume.
+
+{{< trueimage src="/images/SCALE/Apps/InstallSyncthingEnterpriseStorageConfigData1HostPath.png" alt="Syncthing Data1 Storage Settings" id="Syncthing Data1 Storage Settings" >}}
 
-To use an existing dataset created for Syncthing, select **Host Path (Path that already exists on the system)**.
 Enter or browse to the dataset created to populate the **Host Path** field (for example, */mnt/tank/syncthing/config*), then enter or browse to the location of the **data1** dataset to populate the **Host Path** field under the **Mount Path** field.
 
 To add another dataset path inside the container, see [**Storage Settings**](#storage-settings) below for more information.
+Set **Type** to **Host Path (Path that already exists on the system)**, enter **/data1** in **Mount Path**, and then either enter or browse to the path to the **data1** dataset to populate the **Host Path** field.
 
-Click **Install**.
+Accept the rest of the default values in the wizard, then click **Install**.
 The system opens the **Installed Applications** screen with the Syncthing app in the **Deploying** state.
 After installation completes the status changes to **Running**.
 
-{{< trueimage src="/images/SCALE/Apps/SyncthingChartsInstalled.png" alt="Syncthing Installed" id="Syncthing Installed" >}}
+{{< trueimage src="/images/SCALE/Apps/SyncthingEnterpriseAppDeployed.png" alt="Syncthing Installed" id="Syncthing Installed" >}}
 
-Click **Web Portal** on the **Application Info** widget to open the Syncthing web portal to begin configuring folders, devices, and other settings.
+Click **Web Portal** on the **Application Info** widget to open the Syncthing web portal where you can begin configuring folders, devices, and other settings.
 
 {{< trueimage src="/images/SCALE/Apps/SyncthingWebPortalForTrueNAS.png" alt="Syncthing Web Portal for TrueNAS" id="Syncthing Web Portal for TrueNaS" >}}
 
-Secure Syncthing by setting up a username and password.
+### Securing the Syncthing Web UI
+After installing and starting the Syncthing application, launch the Syncthing web UI.
+Go to **Actions > Settings** and set a user password for the web UI.
+
+{{< trueimage src="/images/SCALE/Apps/SyncthingUIActionsMenu.png" alt="Syncthing UI Actions Menu" id="Syncthing UI Actions Menu" >}}
+
+### Using the Syncthing Web Portal for TrueNAS
+
+{{< include file="/static/includes/SyncthingWebPortalInfo.md" >}}
 
-## Understanding Syncthing Settings
-The following sections provide detailed explanations of the settings found in each section of the **Install Syncthing** screen for the Enterprise train app.
+## Understanding TrueNAS Syncthing Wizard Settings
```

Comment:
Not noticing issues with images or names needing updating in this version as much as in the stable version.